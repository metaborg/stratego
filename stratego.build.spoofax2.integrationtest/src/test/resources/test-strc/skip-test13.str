module test13

imports libstratego-lib

strategies
  main = !0; foo
  
  foo : a -> a
    where say(!1)
    with fail
    where say(!3)

