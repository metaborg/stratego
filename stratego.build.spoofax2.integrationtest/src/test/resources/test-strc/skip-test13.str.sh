#!/bin/sh

java -jar ${STRJ} -i test13.str -la stratego-lib -O 0|| exit 1 

rm -f test13.str.actual

javac -cp ${STRJ} test13.java || exit 1

java -cp .:${STRJ} -Xmx1024m test13 > test13.str.actual 2>&1

diff test13.str.expected test13.str.actual || (echo "Result not expected; test failed"; exit 1)
