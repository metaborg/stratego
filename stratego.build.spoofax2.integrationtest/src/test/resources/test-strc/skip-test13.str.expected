1
Internal error: 'with' clause failed unexpectedly in 'foo'
	0
	main_0_0
	foo_0_0
	with_1_1 <==
